packages/loaders/git/tests/test-files/introspection.json

```diff
@@ -963,15 +963,15 @@
       },
       {
         "name": "specifiedBy",
-        "description": "Exposes a URL that specifies the behaviour of this scalar.",
+        "description": "Exposes a URL that specifies the behavior of this scalar.",
```

Comment:
I can certainly drop this, however the vast majority of the code favors this spelling